examples/documentation.html

```diff
@@ -899,6 +899,26 @@ <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist
             desc: "For sunburst only: specifies wether the leaf elements of a parent get the same color as the parent or if they are colored individually.",
             default: "true",
             examples: [true, false]
+        },
+        showLabels: {
+            desc: "For sunburst only: specifies if labels are displayed. The labels are centered on the arcs. The actual output can be adjusted by specifying the 'labelText' function.",
+            default: "false",
+            examples: [true, false]
+        },
+        labelText: {

```

Comment:
hmm... pieChart has labelFormat 

https://nvd3-community.github.io/nvd3/examples/documentation.html#pieChart

that's probably better actually, since this chart is more related to pieChart.